.github/workflows/mkdocs-dev.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
           python-version: 3.x
       - name: Install dependencies
         run: |
+          python -m pip install --upgrade pip setuptools wheel
```

Comment:
installing several dependencies required updating `setuptools` - https://github.com/aquasecurity/trivy/actions/runs/6663370495/job/18109128951
GitHub also recommends updating `setuptools` and `wheel` - https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#installing-dependencies